main/src/ca/uwaterloo/flix/language/ast/ParsedAst.scala

```diff
@@ -925,6 +925,14 @@ object ParsedAst {
       */
     case class FAppend(exp1: ParsedAst.Expression, sp1: SourcePosition, sp2: SourcePosition, exp2: ParsedAst.Expression) extends ParsedAst.Expression
 
+    /**
+      * List expression.
+      * @param sp1  the position of the first character in the `List` keyword.
+      * @param sp2  the position of the last character in the `List` keyword.
```

Comment:
sp2 should come later and span the entire expression (i.e. it should correspond to the last braces in `List#{...}`.

Reply:
I though that too but made it consistent with `Set#{...}` since I assumed it had this pattern for a reason. But that's wrong?

Reply:
Yes, I would say so.

Reply:
Now I am actually not sure. Why is it done like that?